src/ConfigProvider.php

```diff
@@ -6,7 +6,7 @@
  * @license   https://github.com/laminas/laminas-router/blob/master/LICENSE.md New BSD License
  */
 
-declare(strict_types=1);
+declare(strict_types = 1);
```

Comment:
There is no need to change this. Please revert this.
Thanks in advance!

Reference: https://docs.laminas.dev/laminas-coding-standard/v2/coding-style-guide/#3-declare-statements-namespace-and-import-statements

Reply:
not my change `composer cs-fix` did this

Reply:
I have tried to update `laminas/laminas-coding-standard` to 2.0 but it has a lot more problems and I think that's too much and not related to this PR.
if I change it back to `declare(strict_types=1);` then `composer check-cs` complains so not sure what to do :D